README.md

```diff
@@ -20,26 +20,83 @@ The Go library documentation can be found [here][libdocs].
 
 This library supports the following Go implementations:
 
-* Go 1.15
-* Go 1.16
-* Go 1.17
-* Go 1.18
-* Go 1.19
+- Go 1.15
+- Go 1.16
+- Go 1.17
+- Go 1.18
+- Go 1.19
 
 ## Installation
 
-To use twilio-go in your project initialize go modules then run:
+The recommended way to install `twilio-go` is by using [Go modules](https://go.dev/ref/mod#go-get).
 
-```bash
+If you already have an initialized project, you can run the command below from your terminal in the project directory to install the library:
+
+```shell
 go get github.com/twilio/twilio-go
 ```
 
-## Getting Started
+If you are starting from scratch in a new directory, you will first need to create a go.mod file for tracking dependencies such as twilio-go. This is similar to using package.json in a Node.js project or requirements.txt in a Python project. [You can read more about mod files in the Go documentation](https://golang.org/doc/modules/managing-dependencies). To create the file, run the following command in your terminal:
+
+```shell
+go mod init twilio-example
+```
+
+Once the module is initialized, you may run the installation command from above, which will update your go.mod file to include twilio-go.
+
+### Test your installation
+
+Try sending yourself an SMS message by pasting the following code example into a sendsms.go file in the same directory where you installed twilio-go. Be sure to update the accountSid, authToken, and from phone number with values from your Twilio account. The to phone number can be your own mobile phone number.
+
+```go
+package main
+
+import (
+  "encoding/json"
+	"fmt"
+
+	"github.com/twilio/twilio-go"
```

Comment:
I
```suggestion
	twilio "github.com/twilio/twilio-go"
```

We refer to e.g. "twilio.NewRestClient" below without explaining where the 'twilio' comes from

Reply:
Doesn't seem to be necessary, at least based on the behavior of my IDE. If it's a Go convention, sure, but I'd rather not edit every sample unless it's warranted.
<img width="639" alt="image" src="https://user-images.githubusercontent.com/10067791/229952556-2a163150-838d-462b-9ab8-4d6477a31cb9.png">

Reply:
Oh, it's not necessary for sure I just want to avoid confusion. Most of the time the package name matches the import but when they don't match it's not always obvious where it's coming from.

Reply:
I always rename these in my own programs for clarity FYI, I'm not just saying this because it's a tutorial, and it doesn't increase the line count.